src/test/suite/mdbExtensionController.test.ts

```diff
@@ -89,10 +92,10 @@ suite('MDBExtensionController Test Suite', function () {
   });
 
   test('mdb.viewCollectionDocuments command should also work with the documents list', (done) => {
-    const mockOpenTextDocument = sinon.fake.resolves('magna carta');
+    const mockOpenTextDocument: any = sinon.fake.resolves('magna carta');
     sinon.replace(vscode.workspace, 'openTextDocument', mockOpenTextDocument);
 
-    const mockShowTextDocument = sinon.fake.resolves();
+    const mockShowTextDocument: any = sinon.fake();
```

Comment:
The thing with this `any` that when we import Sinon instead of requiring it, lint starts complaining about additional things and `sinon.SinonSpy` that fits here for the `sinon.replace(vscode.workspace, 'openTextDocument', mockOpenTextDocument);` replacement starts erroring with ``Property 'firstArg' does not exist on type 'SinonSpy<any[], any>'`. We can revisit this later and try to find something better than the `any` type, but for now, I think it is ok to use `any`, since before it was without a type anyway.

Reply:
Gotcha sounds good. Will be nice to get rid of some of the `any`s one day. Importing things with their types is a good start. Maybe `firstArg` is an internal thing or something we shouldn't be using 🤔 should be fine to handle it another time.

Reply:
I believe everything is ok with `firstArg`, but we need to figure out how to property create mocks with typescript and Sinon. I was reading the internet about it and looks like it worked with the earlier versions of typescript and start erroring at some point.